src/assets/scripts/client/game/GameController.js

```diff
@@ -31,7 +31,8 @@ const GAME_EVENTS_POINT_VALUES = {
     ILLEGAL_APPROACH_CLEARANCE: -10,
     LOCALIZER_INTERCEPT_ABOVE_GLIDESLOPE: -10,
     NOT_CLEARED_ON_ROUTE: -25,
-    SEPARATION_LOSS: -200
+    SEPARATION_LOSS: -200,
+    NO_TAKEOFF_SEPARATION: -50
```

Comment:
I like tracking these separately like this, but a deal is a deal-- they should probably all have the same penalty value.

Reply:
done